src/main/java/de/espend/idea/shopware/action/generator/ServiceActionUtil.java

```diff
@@ -0,0 +1,68 @@
+package de.espend.idea.shopware.action.generator;
+
+import com.intellij.ide.IdeView;
+import com.intellij.ide.highlighter.XmlFileType;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.actionSystem.DataContext;
+import com.intellij.openapi.actionSystem.LangDataKeys;
+import com.intellij.openapi.application.ApplicationManager;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.ui.Messages;
+import com.intellij.openapi.util.io.StreamUtil;
+import com.intellij.psi.*;
+import com.intellij.psi.codeStyle.CodeStyleManager;
+import org.jetbrains.yaml.YAMLFileType;
+
+import java.io.IOException;
+
+/**
+ * @author Daniel Espendiller <daniel@espendiller.net>
```

Comment:
feel free to add your author :)

Reply:
I have copied the most code from Symfony2 Plugin, so its your code :)